### DISCUSSION TITLE
Issues with runing PHOEBE on Linux Mint 20

### DISCUSSION THREAD

**Discussion by vumaja** [phoebe-project/phoebe2#565] [Questions]
Hi,

a colleague of mine is trying to run PHOEBE on Linux Mint 20 with Python 3.8 and he gets the attached error.
Any ideas how to solve this?
 
![CT_phoebe_error_units](https://user-images.githubusercontent.com/76709806/145496976-85c9ad15-6fb3-4e0f-b106-47a5de4f7a48.png)
![CT_system_info](https://user-images.githubusercontent.com/76709806/145496998-489e368f-de85-4ad8-9116-0a84794f143f.png)

Thanks,
Maja

**Comment by aprsa** [comment]
Hi Maja, I suspect the version of astropy is 5.0? Your colleague will need to downgrade until we account for astropy changes in phoebe.

**Reply by kecnry** [reply]
And you can subscribe to #563 for updates

**Comment by vumaja** [comment]
Hi Andrej,

yes that was it - he has astropy 5.0 !
Thanks a lot,
Maja
Dr. Maja Vuckovic
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Instituto de Física y Astronomía,
Facultad de Ciencias,
Universidad de Valparaíso
Av. Gran Bretaña 1111, Casilla 5030, Valparaíso, Chile
e-mail: ***@***.***
Tel. (+56 32) 250 8304
Fax. (+56 32) 2508135
http://www.dfa.uv.cl
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=

**Comment by vumaja** [comment]
Thanks Kyle!

Dr. Maja Vuckovic
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Instituto de Física y Astronomía,
Facultad de Ciencias,
Universidad de Valparaíso
Av. Gran Bretaña 1111, Casilla 5030, Valparaíso, Chile
e-mail: ***@***.***
Tel. (+56 32) 250 8304
Fax. (+56 32) 2508135
http://www.dfa.uv.cl
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
